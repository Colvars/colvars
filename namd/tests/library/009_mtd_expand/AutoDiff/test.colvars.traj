# step         d                     
           0    1.42467449615693e+01 
           1    1.42470955327846e+01 
           2    1.42476939547291e+01 
           3    1.42483110732975e+01 
           4    1.42492943585138e+01 
           5    1.42503736285750e+01 
           6    1.42515602222333e+01 
           7    1.42528228624295e+01 
           8    1.42541950327222e+01 
           9    1.42556585606080e+01 
          10    1.42572498626192e+01 
          11    1.42590470996093e+01 
          12    1.42605989324750e+01 
          13    1.42621615963684e+01 
          14    1.42637368066359e+01 
          15    1.42651402747233e+01 
          16    1.42662547203391e+01 
          17    1.42670733056824e+01 
          18    1.42680933124264e+01 
          19    1.42689260105063e+01 
          20    1.42698565162903e+01 
