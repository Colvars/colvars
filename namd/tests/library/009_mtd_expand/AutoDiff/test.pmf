# 1
#          0       0.2        76  0

 0.1  0.0191498
 0.3  0.0191498
 0.5  0.0191498
 0.7  0.0191498
 0.9  0.0191498
 1.1  0.0191498
 1.3  0.0191498
 1.5  0.0191498
 1.7  0.0191498
 1.9  0.0191498
 2.1  0.0191498
 2.3  0.0191498
 2.5  0.0191498
 2.7  0.0191498
 2.9  0.0191498
 3.1  0.0191498
 3.3  0.0191498
 3.5  0.0191498
 3.7  0.0191498
 3.9  0.0191498
 4.1  0.0191498
 4.3  0.0191498
 4.5  0.0191498
 4.7  0.0191498
 4.9  0.0191498
 5.1  0.0191498
 5.3  0.0191498
 5.5  0.0191498
 5.7  0.0191498
 5.9  0.0191498
 6.1  0.0191498
 6.3  0.0191498
 6.5  0.0191498
 6.7  0.0191498
 6.9  0.0191498
 7.1  0.0191498
 7.3  0.0191498
 7.5  0.0191498
 7.7  0.0191498
 7.9  0.0191498
 8.1  0.0191498
 8.3  0.0191498
 8.5  0.0191498
 8.7  0.0191498
 8.9  0.0191498
 9.1  0.0191498
 9.3  0.0191498
 9.5  0.0191498
 9.7  0.0191498
 9.9  0.0191498
 10.1  0.0191498
 10.3  0.0191498
 10.5  0.0191498
 10.7  0.0191498
 10.9  0.0191498
 11.1  0.0191498
 11.3  0.0191498
 11.5  0.0191498
 11.7  0.0191498
 11.9  0.0191498
 12.1  0.0191498
 12.3  0.0191498
 12.5  0.0191498
 12.7  0.0191498
 12.9  0.0191498
 13.1  0.0191498
 13.3  0.0191498
 13.5  0.0191498
 13.7  0.0191489
 13.9  0.0188492
 14.1  0.0106064
 14.3  -0
 14.5  0.0157647
 14.7  0.0191026
 14.9  0.0191498
 15.1  0.0191498
