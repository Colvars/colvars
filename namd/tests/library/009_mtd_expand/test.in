colvarsTrajFrequency    1


colvar {
    name d

    upperBoundary 10
    width 0.2
    expandBoundaries yes

    distance {
        group1 { atomnumbers 4 }
        group2 { atomnumbers 99 }
    }
}


metadynamics {
    colvars    d
    hillWeight 0.01
    newHillFrequency 10
    hillWidth      1.2533141373155001  # Old default
}
