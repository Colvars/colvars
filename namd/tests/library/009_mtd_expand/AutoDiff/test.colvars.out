colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  https://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "d"
colvars:   Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.2
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 10
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = on
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "metadynamics" instance.
colvars:   # name = "metadynamics1" [default]
colvars:   # colvars = { d }
colvars:   # zeroStepData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 0 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # hillWeight = 0.01
colvars:   # newHillFrequency = 10
colvars:   # gaussianSigmas =  [default]
colvars:   # hillWidth = 1.25331
colvars:   Half-widths of the Gaussian hills (sigma's):
colvars:   d: 0.125331
colvars:   # multipleReplicas = off [default]
colvars:   # useGrids = on [default]
colvars:   # gridsUpdateFrequency = 10 [default]
colvars:   # rebinGrids = off [default]
colvars:   Metadynamics bias "metadynamics1": Will expand grids when the colvar "d" approaches its boundaries.
colvars:   # writeFreeEnergyFile = on [default]
colvars:   # keepHills = off [default]
colvars:   # keepFreeEnergyFiles = off [default]
colvars:   # writeHillsTrajectory = off [default]
colvars:   # wellTempered = off [default]
colvars:   # biasTemperature = -1 [default]
colvars:   # ebMeta = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Re-initialized atom group for variable "d":0/0. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "d":0/1. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: The final output state file will be "test.colvars.state".
colvars: Opening trajectory file "test.colvars.traj".
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Re-initialized atom group for variable "d":0/0. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "d":0/1. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: The final output state file will be "test.colvars.state".
colvars:   Metadynamics bias "metadynamics1": new upper boundary for colvar "d", at 15.2.
colvars: Saving collective variables state to "test.colvars.state".
